tests/unity/wrap_test/src/fns/.clang-format

```diff
@@ -0,0 +1,3 @@
+{
+    "DisableFormat": true
```

Comment:
Perhaps add a comment explaining why this file exists. (You can just refer to the header file `fns.h` where you give a good explanation)

Reply:
The file looks like JSON-format, which doesn't have support for comments. Are you sure it's safe to add here? It's explained in the header file right next to it.

Reply:
You can add something like this:

`"Comment": This file exists to avoid auto-formatting fns.h. See fns.h for more information.`

But I dont have a strong opinion on this.

Reply:
That depends on how strict validation `clang-format` has on undefined properties, and I don't know it that well. Different tools and versions can produce different outcome, so I'd rather leave it like it is. Unless someone can verify that it's widely known and used without causing issues.